### DISCUSSION TITLE
HW1 part a & c difference?

### DISCUSSION THREAD

**Discussion by Amir-M-Vahedi** [cooperrc/advanced-dynamics#21] [Logistics] [answered]
Hi,

I was wondering what's the difference between part A & part C in HW1.
Because the angle of the pendulum in part A is the variable $\theta$.
Even in part A, I think we need to plot it for both conditions of acceleration because it's not mentioned.

**Comment by kiarash99Naghavi** [comment]
Hi, I have the same question. They both seem similar.

**Comment by cooperrc** [comment]
The main part of question c is to compare the effect of an accelerating reference frame on $\theta(t)$. Putting them both on the same figure to see differences

**Reply by Amir-M-Vahedi** [reply]
Thanks for your answer, but I did the same for the part a.
what is the requested plot for part a? (just plotting without acceleration for example?)

**Reply by SentientBiped** [reply, marked as answer]
Yes, just the non-accelerating reference frame result for part a.
